packages/quantic/cypress/e2e/facets-2/numeric-facet/numeric-facet.cypress.ts

```diff
@@ -429,17 +429,4 @@ describe('quantic-numeric-facet', () => {
       });
     });
   });
-
-  describe('with default numeric facet testing accessibility', () => {
```

Comment:
Will we fix these later on? If so it's probably better to .skip them no?

Reply:
Yes, skipping them is a better idea, let me fix that👌